src/olympia/files/file_viewer.py

```diff
@@ -1,512 +0,0 @@
-import codecs
```

Comment:
you've left the file there as 0 byte file?

Reply:
Ah, thanks. That was unintentional. I kept deleting the code, then deleted all empty files and apparently missed two.